src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -518,6 +518,42 @@
                     }
                 }
             ]
+        },
+        {
+            "path": "/v1/debug/bundle/files/{filename}",
+            "operations": [
+                {
+                    "method": "DELETE",
+                    "summary": "Deletes from disk the zip file containing the bundle",
+                    "type": "string",
+                    "nickname": "delete_debug_bundle",
+                    "produces": [
+                        "application/json"
+                    ],
+                    "parameters": [
+                        {
+                            "name": "filename",
+                            "in": "path",
+                            "required": true,
+                            "type": "string"
+                        }
+                    ],
+                    "responses": {
+                        "200": {
+                            "description": "Successfully deleted the debug bundle"
+                        },
+                        "404": {
+                            "description": "The debug bundle is running but not yet ready"
+                        },
+                        "410": {
+                            "description": "The debug bundle is not running or it is not on disk"
+                        },
+                        "500": {
+                            "description": "Internal Server error"
+                        }
+                    }
+                }
+            ]
```

Comment:
Similar question to the one above, can we add this to the already created list of `operations` for `/v1/debug/bundle/files/{filename}`?